src/utils/access_token.ts

```diff
@@ -76,7 +77,7 @@ async function provision(): Promise<string> {
       wait("").start().warn(
         "Unexpected error while trying to open the authorization URL in your default browser. Please report it at https://github.com/denoland/deployctl/issues/new.",
       );
-      wait({ text: "", indent: 3 }).start().fail(error.toString());
+      wait({ text: "", indent: 3 }).start().fail(stringifyError(error));
```

Comment:
Can you check if this change breaks the output format? IIRC it was sensible to multiline error messages.

Reply:
If an error is formatted in a multi-line format, it is printed like this:

<img width="1088" alt="CleanShot 2024-10-18 at 20 29 43@2x" src="https://github.com/user-attachments/assets/abf21aeb-147c-4c29-8d59-35367aa980ce">

This error instance is created as follows:

```ts
    const e1 = new Error("This is\na multiline\nerror");
    const e2 = new Error("e2", { cause: e1 });
    throw e2;
```

I am not sure if I understand your concern, but does this look concerning to you?

Reply:
It's purposedly identated and kept in a single line because it is not a hard error, the program continues. If the error breaks this identation then it breaks the output layout.

Reply:
Now `stringifyError(error)` returns a simple one-line error message while `stringifyError(error, { verbose: true })` generates a full detailed output.